### DISCUSSION TITLE
Using a shared Google Drive folder confuses media items

### DISCUSSION THREAD

**Discussion by stepstra** [sircharlo/meeting-media-manager#1285] [Q&A] [answered]
Our setup until today:
We use a master Google Drive account for all things related to our congregation. In the KH M3 is set up to use a shortcut to that cloud folder via the Google Drive sync client, which mirrors the actual folder to the hard drive.

This master folder is shared to two other brothers private Google Drive accounts. These also use the sync client on their desktop PCs at home and M3 is set up to use these as well (with the same folder naming and settings).

Now: when one brother prepares the media for a week the sync client perfectly uploads these files to the master folder. But when the other brother presses update media folder in M3 on his PC all previously uploaded media by the first brother "disappears" - actually it is not being deleted but moved back to the private account of the first brother in the main folder; all files without any structure. Can't make heads or tails out of this.

**Comment by mtdvlpr** [comment]
Hmm, I have no experience with this setup, but I do know that M3 sometimes doesn't have access rights to local Google Drive folders and can't therefore read and/or write to it.

**Reply by mtdvlpr** [reply]
@stepstra, are there any M3 warnings when performing these actions?

**Reply by stepstra** [reply]
None that I know of. I also looked up the activity history inside the master Google Drive - but I can't see the movement there.

**Comment by mtdvlpr** [comment]
@e-Feitosa, don't you have experience with such a setup? Experienced anything like this before?

**Reply by e-Feitosa** [reply]
@mtdvlpr , i use Google Drive too, similar to what @stepstra talked about. And this situation mentioned actually occurs. Here we decided to leave the M3 without the option to download the media automatically, so when the sound operator opens the M3, he already checks that the media are OK. What do we notice? Google Drive takes time to synchronize the media that were sent, and therefore, the M3 "erases" by downloading the media again.

**Comment by mtdvlpr** [comment, marked as answer]
Alright, so basically the problem is M3 is downloading media into the Google Drive folder before Google Drive has synced all the media. Since it's the same media with the same filenames, Google Drive doesn't know what to do with this conflict and weird things happen.

**Reply by mtdvlpr** [reply]
That's why using a WebDAV server is preferred
